submissions/usides/html-css-popup/css/main.css

```diff
@@ -0,0 +1,376 @@
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  -webkit-box-pack: end;
+      -ms-flex-pack: end;
+          justify-content: flex-end;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  margin: 0.3rem;
+  -webkit-box-flex: 0;
+      -ms-flex-positive: 0;
+          flex-grow: 0;
+  text-decoration: none;
+  padding: 0.2rem 0.9rem;
+  font-size: 0.9rem;
+  border-radius: 0.4rem;
+  -webkit-transition: background-color 0.3s;
+  transition: background-color 0.3s;
+}
+
+nav ul .menu .menu-popup .menu-popup-item:focus {
+  background-color: rgba(255, 255, 255, 0.2);
+}
+
+nav ul .menu .menu-popup .menu-popup-item p {
+  color: #fff;
+  font-family: 'Rajdhani', sans-serif;
+}
+
+nav ul .menu .menu-popup .menu-popup-item img {
+  margin-bottom: 0.4rem;
+}
+
+nav ul .menu .menu-popup .hidden {
+  display: none;
+}
+
+nav ul .menu .menu-popup .location img,
+nav ul .menu .menu-popup .video img {
+  width: 90%;
+}
+
+nav ul .menu .menu-popup .calendar img,
+nav ul .menu .menu-popup .audio img {
+  width: 75%;
+}
+
+nav ul .menu .menu-popup .human img,
+nav ul .menu .menu-popup .cat img {
+  width: 80%;
+}
+
+nav ul .user-logo {
+  display: block;
+  max-width: 2.5rem;
+  border-radius: 50%;
+  border: 3px solid #fff;
+  -webkit-transition: all 0.2s;
+  transition: all 0.2s;
+}
+
+nav ul .user-logo:focus {
+  border: 3px solid #423e3e;
+}
+
+main {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+          transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+main form h1 {
+  font-family: 'Russo One', sans-serif;
+  color: #423e3e;
+  -webkit-text-stroke: 0.25rem #fff;
+  font-size: 3.7rem;
+  letter-spacing: 0.1rem;
+}
+
+main form .search-panel .input-field {
+  min-width: 15rem;
+  min-height: 2.7rem;
+  border-radius: 2rem;
+  background-color: #0000005b;
+  border: 0.2rem solid #fff;
+  font-family: 'Red Hat Display', sans-serif;
+  font-size: 1.5rem;
+  color: #fff;
+  -webkit-transition: all 300ms;
+  transition: all 300ms;
+  text-align: center;
+}
+
+main form .search-panel .input-field:focus {
+  min-width: 35rem;
+  text-align: left;
+  text-indent: 1.5rem;
+}
+
+main form .search-panel .input-field::-webkit-input-placeholder {
+  color: #fff;
+}
+
+main form .search-panel .input-field:-ms-input-placeholder {
+  color: #fff;
+}
+
+main form .search-panel .input-field::-ms-input-placeholder {
+  color: #fff;
+}
+
+main form .search-panel .input-field::placeholder {
+  color: #fff;
+}
+
+main form .search-panel .submit-button {
+  display: block;
+  margin-top: 1rem;
+  min-width: 5rem;
+  min-height: 5rem;
+  background-image: url(../img/paw.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 1rem auto 0;
+  cursor: pointer;
+  padding: 0;
+}
+
+main form .search-panel .submit-button:focus {
+  background-color: #ffffff91;
+  border-radius: 50%;
+}
+
+footer {
```

Comment:
In CSS we can push` footer` to the bottom without `position: absolute;` in several ways.
In this case, you can leave it, but in a real project this solution not pass code review.
Change it or leave it it's up to you, just let me know about your decision.)

Reply:
Thanks for this info. I will fix it.